static/js/narrow.js

```diff
@@ -272,11 +272,14 @@ exports.activate = function (raw_operators, opts) {
         });
     }
 
-    if (filter.has_operator("is") && filter.operands("is")[0] === "private"
-        || filter.has_operator("pm-with") || filter.has_operator("group-pm-with")) {
-        compose.update_closed_compose_buttons_for_private();
+    if (filter.has_operator("is") && (filter.operands("is")[0] === "private"
```

Comment:
I still think this is prone to error, and it would be better to have the `if` go in the other direction. I.e. "if in a stream narrow, then .. else ..".

Reply:
This was a lot convoluted initially when we had different text for PM narrows and I also kind of followed what we had earlier.
Looking at it now, it can be simplified a lot, and I've made the change. Thanks for bringing this up!